src/frontend/components/smallVideoCard/smallVideoCard.js

```diff
@@ -1,19 +1,60 @@
 import Moment from 'react-moment';
```

Comment:
Here, I am unable to click on text title of video to go to single video page. maybe the link / navigate could encompass the entire card than just the thumbnail ? just nitpicking ux